### PR TITLE
Wip 7 , 5 and 6

### PR DESCRIPTION
**Pull request by Weintraubm**

### CODE REVIEW

File: exercise7-5.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+because it only has one primary key
```
Comment (smattingly) [inline exercise7-5.txt:1]: Every table has one primary key, by definition. I think you've got the idea, but haven't expressed in correctly.

**Review by smattingly — CHANGES_REQUESTED** [review]
Most of the review comments are just for feedback. The only thing I'd like you to change is 7-5. Because this PR is proper subsets of multiple exercise sets, I'm unclear what if anything should be checked off in Sakai, but I'll figure that out.

File: exercise7-5.txt [hunk 2]
```diff
@@ -1 +1 @@
-because it only has one primary key
\ No newline at end of file
+because it has no trivial keys and only needs one reference to connect to other tables rather than multiple references
```
Comment (smattingly) [inline exercise7-5.txt:1]: Again, I think you have the idea. With proper terminology: the primary key is a single field (not a compound key), therefore no proper subsets of the key, therefore no partial dependencies can exist.

**Review by smattingly — APPROVED** [review]
👍

**Review by Weintraubm — COMMENTED** [review]
@smattingly

**Review by smattingly — APPROVED** [review]
8-1 looks good. Would you please merge (and thus close) this PR? Then I can just track new work by watching commits. It has become difficult to keep track of things when mixed together this way-- I should have planned better.

**Pull request merged**